MoneySplitter.Infrastructure/INavigationManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MoneySplitter.Models;
+
+namespace MoneySplitter.Infrastructure
+{
+    public interface INavigationManager
+    {
+        void NavigateToShellView(UserModel userModel);
```

Comment:
ShellViewModel should get userdata by itself (From membership service). It not need additional parameters

Reply:
moved LogiModel

Reply:
fixed. removed parametr